src/app.ts

```diff
@@ -190,22 +191,25 @@ export function newApp(
       API_CLIENT
     );
 
+    // Create the Notification Service
+    // TODO: This call may throw an error
```

Comment:
you may consider a factory function instead of constructor that return an `Either<NotificationService>` calling `tryCatch2v`

Reply:
Fixed in fd811c7